lib/shelley/src/Cardano/Wallet/Shelley/Launch/Cluster.hs

```diff
@@ -416,9 +418,17 @@ data ClusterEra
     | MaryHardFork
     deriving (Show, Read, Eq, Ord, Bounded, Enum)
 
+-- | Convert @ClusterEra@ to a @ApiEra@.
+clusterToApiEra :: ClusterEra -> ApiEra
+clusterToApiEra = \case
```

Comment:
Could replace `ClusterEra` with `ApiEra`, but I think this is fine. Maybe they diverge sometime.